axlearn/common/flash_attention/layer.py

```diff
@@ -129,6 +131,13 @@ def _compute_attention(
         k_proj = self._repeat_kv_heads(k_proj)
         v_proj = self._repeat_kv_heads(v_proj)
 
+        if attention_logit_biases is not None and segment_ids is not None:
```

Comment:
Where does this limitaiton come from?

Reply:
Please see the explanation above. Thoughts?